packages/teleport/src/Player/Player.story.tsx

```diff
@@ -40,3 +40,20 @@ export const Player = () => {
     </Router>
   );
 };
+
```

Comment:
lets fix the `Player` story, it's broken b/c it expects a `recordingType` in url, we also should probably make another story to see rendering of both types of players

Reply:
Updated. FYI the story surfaces this known issue: https://github.com/gravitational/webapps/issues/574